app/labor/badge_rewarder.rb

```diff
@@ -1,8 +1,17 @@
 module BadgeRewarder
   def self.award_yearly_club_badges
-    award_one_year_badges
-    award_two_year_badges
-    award_three_year_badges
+    years = 3
+    (1..years).each do |i|
+      message = "Happy DEV birthday! Can you believe it's been #{i} years already?!"
```

Comment:
The pluralization of `year` will be wrong for the first year.

> Happy DEV birthday! Can you believe it's been 1 years already?!

```suggestion
      message = "Happy DEV birthday! Can you believe it's been #{i} #{"year".pluralize(i)} already?!"
```

Reply:
@abraham I didn't even think about this, thank you!